src/presentation/ui/component/util/toast.templ

```diff
@@ -22,18 +22,27 @@ script ToastGlobalState() {
 	});
 
 	document.addEventListener('htmx:afterRequest', (event) => {
-		const contentType = event.detail.xhr.getResponseHeader("Content-Type");
+		const httpResponseObject = event.detail.xhr;
+
+		const isTokenResponse = httpResponseObject.getResponseHeader(
+			"X-Is-Token-Response",
+		);
+		if (isTokenResponse === 'true') {
+			return;
+		}
```

Comment:
Ok, here's the header. Dude, remember what we discussed during the call? Instead of using a X-Is-Token-Response which is obviously an edge case, just have a generic header, like "X-Ignore-Toast". That way you can use that same thing for other endpoints in the future.